FastBitSet.js

```diff
@@ -118,7 +118,7 @@ FastBitSet.prototype.trim = function (index) {
   while (nl > 0 && this.words[nl - 1] === 0) {
     nl--;
   }
-  this.words = this.words.slice(0, nl);
+  this.words.length = nl;
```

Comment:
slice might create a copy, but setting length certainly will not.

Reply:
> slice might create a copy, but setting length certainly will not.

At a semantic level, what is hopefully true is that you save on the creation of an object, which is not a negligible gain... but the data itself is likely copied at least some of the time, if only during memory compaction, as part of the garbage collection. Address space is linear. If you just downsize arrays, you won't reduce memory usage. 

Whether it is more efficient would be best assessed experimentally but if someone relies on trim to be highly efficient... ?

(Note that I am not arguing with the change: it is obviously better code.)

Reply:
thank you!